templates/modern/src/nav.ts

```diff
@@ -73,7 +73,9 @@ export async function renderNavbar(): Promise<NavItem[]> {
       return []
     }
 
-    const navUrl = new URL(navrel.replace(/.html$/gi, '.json'), window.location.href)
+    const navUrl = navrel.startsWith('/')
```

Comment:
`document.baseURL` is `window.location.href` when `<base>` is not set. Does this work?

`const navUrl = new URL(navrel.replace(/.html$/gi, '.json'), document.baseURL)`

Reply:
> document.baseURL is window.location.href when <base> is not set. Does this work?

Sorry for the confusion.
This PR intended to fix toc loading issue when **custom** `<base>` tag is used on HTML file.

If `<base>` tag is not exists on HTML file.
There is no behavior differences.